teuthology/calc.py

```diff
@@ -0,0 +1,31 @@
+from teuthology.suite.build_matrix import build_matrix
+from teuthology.suite.build_matrix import _get_matrix
+
+def  main(args):
```

Comment:
Needs doc string

Reply:
@vasukulkarni Not sure I understand, the docstring is in the script/calc.py already

Reply:
The code moved to teuthology/suite.py